src/components/Composer.vue

```diff
@@ -174,7 +174,7 @@
 	</div>
 	<div v-else class="emptycontent">
 		<h2 v-if="!isReply">{{ t('mail', 'Message sent!') }}</h2>
-		<h2 v-else>{{ t('mail', 'Reply sent!') }}</h2>
+		<h2 v-else>{{ t('mail', 'Message sent!') }}</h2>
```

Comment:
sure, but in that case we don't need the if-else, do we? :)

Reply:
We need one for new message v-if, and one for everything else, v-else.

Reply:
Look again. It's the same element in both cases: `<h2>{{ t('mail', 'Message sent!') }}</h2>`

Reply:
Please check isReply function. If we remove v-else, reply and forward will not have a feedback message.

Reply:
Let me rephrase. Instead of

```html
<h2 v-if="!isReply">{{ t('mail', 'Message sent!') }}</h2>
<h2 v-else>{{ t('mail', 'Message sent!') }}</h2>
```

we can simply do

```html
<h2>{{ t('mail', 'Message sent!') }}</h2>
```

`v-if` and `v-else` render the same element.

Reply:
dammit